tests/greenio_test.py

```diff
@@ -1061,3 +1061,23 @@ def test_greenpipe_read_overwrite():
 
         actual = tests.read_file(f.name)
         assert actual == new_data
+
+
+def test_greenpipe_write_plus():
+    excepted = "write with mode=w+"
```

Comment:
typo

```suggestion
    expected = "write with mode=w+"
```

exCept close to but/different. exPect close to prediction/wish.

Example dataset: [1,3,5,7,2]
- all numbers are odd except 2; `try/except/finally`; `raise Exception()`
- after 1,3,5,7 we expect 9; 2 is unexpected; expectation is broken

Reply:
I misspelled words of excepted and expected,I had correct it.Forgive my poor english.